driver-scala/src/main/scala/org/mongodb/scala/model/Aggregates.scala

```diff
@@ -488,7 +505,8 @@ object Aggregates {
    *
    * @param operator A search operator.
    * @param options Optional `\$search` pipeline stage fields.
-   * Specifying `null` is equivalent to specifying `SearchOptions.defaultSearchOptions`.
+   * Specifying `null` is equivalent to specifying `SearchOptions.defaultSearchOptions`
```

Comment:
I missed this last time - passing `null` in Scala is a big no no, so I'd remove this line.

The idiom in scala would be `Option[SearchOptions]` to signify optional / nullable values.

Reply:
Done.